#start joint configuration
float64[] start
#goal joint configuration
float64[] goal

string[] joint_names

#group name of set of joints (e.g. right_arm)
string group_name

#time planner(s) can take before giving up
duration allowed_planning_time
---
#planned path: if empty, then planning failed
Float64Array[] path

Status status
---
